### PR TITLE
Harden new distributed_actor_protocol_call_resilient_lib test for lib search paths

### PR DESCRIPTION
**Pull request by ktoso**
This hardens the test for envs where the libs are not found without explicitly linking.

Thanks @xedin for the hint how to solve this.

resolves rdar://132714652

### CODE REVIEW

**Comment by ktoso** [comment]
@swift-ci please smoke test

**Comment by ktoso** [comment]
@swift-ci please test

**Review by xedin — APPROVED** [review]

File: test/Distributed/Runtime/distributed_actor_protocol_call_resilient_lib.swift [hunk 1]
```diff
@@ -52,7 +52,11 @@
 // RUN:     -o %t/a.out
 
 // RUN: %target-codesign %t/a.out
```
Comment (xedin) [inline test/Distributed/Runtime/distributed_actor_protocol_call_resilient_lib.swift:54]: @ktoso you also need to codesign the dylibs that are using in %target-run

Comment (ktoso) [reply test/Distributed/Runtime/distributed_actor_protocol_call_resilient_lib.swift:55]: Oh, thank you, done now

**Comment by ktoso** [comment]
@swift-ci please test

**Pull request merged**